pallets/loans/src/lib.rs

```diff
@@ -198,19 +194,27 @@ pub mod pallet {
 		type NonFungible: Transfer<Self::AccountId>
 			+ Inspect<Self::AccountId, CollectionId = Self::CollectionId, ItemId = Self::ItemId>;
 
+		/// The PoolId type
+		type PoolId: Member + Parameter + Default + Copy + HasCompact + MaxEncodedLen;
```

Comment:
Ok, so the downside of leaving the trait cleaned, is the addition of an associated type to the config pallet that wants to use such a trait. 👍🏻

Ideally would be great to add those bounds in the `Config` where clause. I've tried before, but it requires to be propagated to all impl blocks that uses `Config`, and it comes with a lot of boilerplate and the errors are quite obfuscated. So I think this is the less painful way 👍🏻 

Thanks to do the test Nuno!